### DISCUSSION TITLE
[BUG] When trying to select value of index

### DISCUSSION THREAD

**Discussion by aaronchishui** [stephanediondev/elasticsearch-admin#72] [General]
I have a problem. Why trying to show source value of docs in index: "transaction". The error is: `Failed to execute 'querySelector' on 'Element': '#modalMBS_8070.0_20231011' is not a valid selector.`
![image](https://github.com/stephanediondev/elasticsearch-admin/assets/2629051/7ff291bc-531b-4a7d-bf80-2eaa6f091356)
The others indexes still working.
Help me please.

**Comment by stephanediondev** [comment]
Hi

It's due to the "_id" ES field used for the modal ID attribute: there is a dot in "MBS_8070.0_20231011"

It's updated: now the modal IDs are indexed (modal0, modal1, modal2 ...)

Commit https://github.com/stephanediondev/elasticsearch-admin/commit/4b5152a0cd0afb4753d93286f8314492a17e784b

Release 1.81
https://github.com/stephanediondev/elasticsearch-admin/releases/tag/v1.81

**Reply by ducanh2110** [reply]
Thanks for reply fast. I will pull new image

**Reply by ducanh2110** [reply]
@stephanediondev hi bro can I ask how to export full sources data include runtime fields? Thanks

**Comment by stephanediondev** [comment]
Hi 

New image available

Now in search results and search export you have the fields from mappings/properties AND mappings/runtime

https://github.com/stephanediondev/elasticsearch-admin/commit/54ac172f6f9d787ee398b28125419a5fb12d4bd6

**Reply by ducanh2110** [reply]
Thanks so much. I will test now
